src/becca/entities/bnote.ts

```diff
@@ -450,7 +450,7 @@ class BNote extends AbstractBeccaEntity<BNote> {
         );
     }
 
-    getAttributeCaseInsensitive(type: string, name: string, value: string | null) {
+    getAttributeCaseInsensitive(type: string, name: string, value?: string | null) {
```

Comment:
nitpick: if value is optional the type union with null is not needed

Reply:
@Micosilent , this is done due to this:

```js
value = value ? value.toLowerCase() : null;
```

This piece of code reuses the `value` argument and assigns a `null` to it. The optional type only handles `undefined`.

Reply:
Should we then not assign undefined instead of null?

Reply:
@alexpietsch , it would make sense however I wanted to keep as close as possible to the original implementation to simplify the PRs and to avoid any accidental mistakes from misunderstanding the code.